### PR TITLE
Revert "Use throng for node clustering and worker restarts (#157)"

### PR DESCRIPTION
**Pull request by schneems**
This reverts commit 02f5712d4e76b81f50f9f3df07ead16f4c224393.

In manual testing, it was found that local debugging with a debugger was difficult with a clustered server. The debugger will attach to one process, but not all of them. When that happens, only requests that go to the process with the debugger attached will have the expected functionality. 

In the long term, we can change the logic to boot with no workers in the local development mode, or explore other solutions. In the short term, we are rolling back the clustering changes.

GUS-W-9964160

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -15,10 +15,11 @@
         "jsforce2": "npm:jsforce@^2.0.0-beta.8",
         "throng": "^5.0.0",
         "whatwg-mimetype": "^3.0.0",
+        "whatwg-mimetype": "^3.0.0",
```
Comment (edmorley) [inline npm-shrinkwrap.json:18]: Is this a rebase issue?

Comment (schneems) [reply npm-shrinkwrap.json]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0] - 2021-10-01
+
+ - Revert NodeJS clustering, manage workers via throng ([#157](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/192))
```
Comment (edmorley) [inline CHANGELOG.md:11]:
```suggestion
 - Revert NodeJS clustering, manage workers via throng ([#192](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/192))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,7 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Convert package-lock.json to npm-shrinkwrap.json for deterministic dependencies ([#165](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/165))
 - Rename apexClassId -> apexId and apexClassFQN -> apexFQN to support Apex classes and triggers.
-- Introduce NodeJS clustering, manage workers via throng ([#157](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/157))
+ - Introduce NodeJS clustering, manage workers via throng ([#157](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/157))
```
Comment (edmorley) [inline CHANGELOG.md:21]:
```suggestion
- Introduce NodeJS clustering, manage workers via throng ([#157](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/157))
```

**Review by edmorley — APPROVED** [review]

**Pull request merged**